ycmd/completers/python/jedi_completer.py

```diff
@@ -22,23 +22,40 @@
 from ycmd.utils import ToUtf8IfNeeded
 from ycmd.completers.completer import Completer
 from ycmd import responses
+from ycmd import utils
 
-try:
-  import jedi
-except ImportError:
-  raise ImportError(
-    'Error importing jedi. Make sure the jedi submodule has been checked out. '
-    'In the YouCompleteMe folder, run "git submodule update --init --recursive"')
+import urlparse
+import requests
+import logging
+
+import os
+import sys
+
+
+PATH_TO_PYTHON = sys.executable
+PATH_TO_JEDIHTTP = os.path.join(
+  os.path.abspath( os.path.dirname( __file__ ) ),
+  '..', '..', '..', 'third_party', 'JediHTTP', 'jedihttp' )
```

Comment:
I see. I was mostly thinking of compatibility, but since they use the same string anyway it's probably not very important. I guess the advantage would be if some platform used something other than `..` it will just work (given that Python supports that platform of course).